plugin/trino-hive-hadoop2/bin/run_hive_abfs_access_key_tests.sh

```diff
@@ -26,7 +26,8 @@ stop_unnecessary_hadoop_services
 # run product tests
 pushd $PROJECT_ROOT
 set +e
-./mvnw -B -pl :trino-hive-hadoop2 test -P test-hive-hadoop2-abfs-access-key \
+MAVEN_TEST="${MAVEN_TEST:--B --strict-checksums -Dmaven.javadoc.skip=true -Dmaven.source.skip=true -Dair.check.skip-all}"
```

Comment:
Actually, let's not change these defaults, as it's different than simply passing the test flags. Since this would only affect running the scripts manually, which likely no one does, it doesn't have any impact and makes things more complicated.

Reply:
I removed skipping Javaodc. I'm not sure I understood the second comment. Changes here are needed to enable GIB. I wanted to avoid duplicating the whole `MAVEN_TEST` line in every script here but have some sane defaults. Is it better to hardcode all Maven options (and add -Pgib)?

Reply:
Previously, if you ran the script directly, it the argument would be `-B`. You added the default value via the Bash `:-` expansion to add many more arguments. We don't need to do that, as that is out of scope for this change.

Reply:
Per my other comment, the minimal change to pass `MAVEN_TEST` would be
```shell
./mvnw ${MAVEN_TEST:--B} -pl :trino-hive-hadoop2 test -P test-hive-hadoop2-abfs-access-key \

Reply:
All clear now, thanks!